src/app/components/common/node-editor/custom-node/custom-node.component.ts

```diff
@@ -68,14 +68,14 @@ export class CustomNodeComponent implements OnChanges {
   filter = { pluginName: '', enabled: 'false', name: '', color: '', pluginVersion: "" }
   isServiceNode: boolean = false;
   subscription: Subscription;
-  addFilterSubscription: Subscription;
+  // addFilterSubscription: Subscription;
   pluginName = '';
   isFilterNode: boolean = false;
   destroy$: Subject<boolean> = new Subject<boolean>();
   fetchedTask;
   fetchedService;
-  showPlusIcon = false;
-  showDeleteIcon = false;
+  // showPlusIcon = false;
+  // showDeleteIcon = false;
```

Comment:
Please remove commented code from here and other files. And there are some unused imports in this file, remove those as well.